runtime/altair/src/lib.rs

```diff
@@ -1320,50 +1318,23 @@ impl pallet_rewards::Config<pallet_rewards::Instance2> for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 }
 
-// PoolSystem & Loans
-
 parameter_types! {
 	pub const MaxActiveLoansPerPool: u32 = 300;
-	pub const MaxRateCount: u32 = MaxActiveLoansPerPool::get();
-	pub const MaxCollectionSize: u32 = MaxActiveLoansPerPool::get();
+	pub const MaxRateCount: u32 = 300; // See #1024
```

Comment:
I would forget to comment on that!

As far as I know, `MaxRateCount` should be greater than `MaxActiveLoansPerPool`. If not, if one pool reaches `MaxActiveLoansPerPool,` there are no more rates available for other pools. That's why I maintain the value (by now) but disassociate it from the other parameter.

Reply:
I've comment this into issue #1024 to track this change

Reply:
IIRC we restrict the rate to be between 100.00 and 0, so we have a max amount of rates equal to 10001. 

What would be the weight of that? Shouldn't we just allow that maximum?

Reply:
Ohh, I didn't think about that inherent limitation from the available rates. 

Then, yes, we can use that value if we're sure that computing power over 10000 elements does not exceed the block limit. We would need to perform a benchmark first. 

I would go with this for another PR, where we first make a benchmark for the worst case. Do you agree? Actually, it can be an update for the #1024 issue.

Reply:
We can even create an utility method for it, which can be called from the runtime as follows: 
```rust
type MaxCountRate = pallet_interest_accrual::Pallet::<Runtime>::get_max_rate_count(0.25, MAXIMUM_BLOCK_WEIGHT)
``` 
returning the exact value to not overpass the 25% of the block limit.

Reply:
That would be great!! Agreed with all above!